safe-global/web-core
# fix: tx filter improvements

## What it solves

Transaction filter improvements.

## How this PR fixes it

The transaction filter UX has been adjusted to include the following:

1. Applying a filter hides the form.
2. The date format has been changed to dd/MM/yyyy.
3. Number of results is shown. (This will vastly changed with the rebrand so it is a basic implementation atm).
4. Styling adjustments to match that of `safe-react`.

A small refactor was also included to `_addDateLabels`.

## How to test it

1. Apply a filter. Observe that the form hides.
2. Enter a date. Observe that it is now dd/MM/yyyy format.
3. Apply a filter. Observe the number of results under the button.
4. Compare the form with that of `safe-react`. The form labels and divider should now match.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/187628030-2d7d6237-af51-4b2c-8729-27be85e4234c.png)

![image](https://user-images.githubusercontent.com/20442784/187628129-40d9b990-bc72-4e5a-92da-47c323c82e8b.png)

![image](https://user-images.githubusercontent.com/20442784/187628957-7fbeb16d-72e2-40dd-a230-08d97399d09a.png)
